city/crud.py

```diff
@@ -0,0 +1,23 @@
+from sqlalchemy import select, insert, update, delete
+from sqlalchemy.ext.asyncio import AsyncSession
+
+from city import models, schemas
+
+
+async def get_all_cities(db: AsyncSession):
+    query = select(models.City)
+    result = await db.execute(query)
+    return result.scalar().all()
```

Comment:
The method `scalar().all()` is incorrectly used here. It should be `scalars().all()` to correctly retrieve all results from the query.